LICENSE-3rdparty.csv

```diff
@@ -1,2 +0,0 @@
-Component,Origin,License,Copyright
-import,io.opentracing,Apache-2.0,Copyright 2016-2017 The OpenTracing Authors
```

Comment:
I don't think we can remove this. In addition to that, we should probably add notice about datadog's copyright here

Reply:
reverted

Reply:
the old license as well (just read your comment on Slack), thanks 👍